library/common/jni/jni_interface.cc

```diff
@@ -873,18 +873,41 @@ Java_io_envoyproxy_envoymobile_engine_JniLibrary_registerStringAccessor(JNIEnv*
   return result;
 }
 
+static void jvm_on_track(envoy_map events, const void* context) {
```

Comment:
definitely not familiar with how we are dealing with memory in Java bridging code etc so I may be wrong here but I think that we are missing the release of `events` memory?

Reply:
it sounds that it should be the case, but im not sure, given my limit on memory mamangement.

for this method i followed the way `jvm_on_log` method releases its resources

Reply:
Correct, it will need to be released somehow. In other code (iirc) the native_x_to_y functions handle it, but that seems to not be the case here. Regardless, `release_envoy_map` needs to be called when we're done with it.

Reply:
i see 👌